ddtrace/llmobs/_llmobs.py

```diff
@@ -685,14 +685,24 @@ def submit_evaluation(
         if not label:
             log.warning("label must be the specified name of the evaluation metric.")
             return
+
         if not metric_type or metric_type.lower() not in ("categorical", "numerical", "score"):
-            log.warning("metric_type must be one of 'categorical', 'numerical', or 'score'.")
+            log.warning("metric_type must be one of 'categorical' or 'score'.")
             return
+
+        metric_type = metric_type.lower()
+        if metric_type == "numerical":
+            log.warning(
+                """The evaluation metric type 'numerical' is deprecated. Use 'score' instead. """
+                """Converting `numerical` metric to `score` type."""
```

Comment:
```suggestion
                "The evaluation metric type 'numerical' is deprecated. Use 'score' instead. "
                "Converting `numerical` metric to `score` type."
```
Do we need the triple quotes?

Reply:
We do not